<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LANchat</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>100</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.488, y1:1, x2:0.509, y2:0.00284091, stop:0 rgba(99, 120, 133, 255), stop:1 rgba(136, 184, 184, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>-1</y>
     <width>250</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.488, y1:1, x2:0.509, y2:0.00284091, stop:0 rgba(128, 155, 172, 255), stop:1 rgba(177, 220, 220, 255))</string>
   </property>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>35</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(88, 120, 132);
color: rgb(234, 255, 220);
font: 12pt &quot;Microsoft YaHei UI&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>搜索</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="contactListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>99</y>
      <width>251</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>-1</y>
     <width>1100</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0.488, y1:1, x2:0.509, y2:0.00284091, stop:0 rgba(186, 207, 220, 255), stop:1 rgba(206, 232, 232, 255))</string>
   </property>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.488, y1:1, x2:0.509, y2:0.00284091, stop:0 rgba(142, 171, 188, 255), stop:1 rgba(172, 218, 218, 255));</string>
    </property>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>180</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;
background-color: #1472D0;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>961</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imageButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>180</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/res/5c4803c90d7b69ecfb0d4366eca3dac7.png);
color: rgba(253, 253, 253, 0);
 border: none;
     
      background-repeat: no-repeat;
     background-position: center;
      background-origin: content;
      background-size: stretch;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>180</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/res/734c92fe89a49c68501d96970fe11111.png);
color: rgba(255, 255, 255, 0);
 border: none;
    
     background-repeat: no-repeat; 
     background-position: center; 
  background-size: cover;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <zorder>textEdit</zorder>
    <zorder>sendButton</zorder>
    <zorder>imageButton</zorder>
    <zorder>fileButton</zorder>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1000</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);
font: 12pt &quot;Microsoft YaHei UI&quot;;
QLabel { color: gray; }</string>
     </property>
     <property name="text">
      <string>联系人</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close1Button">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(255, 0, 4);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minButton">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(0, 0, 255);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="maxpageButton">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color:rgb(0, 170, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="chatBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>961</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
